### DISCUSSION TITLE
SSH Protocol Handler

### DISCUSSION THREAD

**Discussion by SaveEnergy** [microsoft/terminal#18003] [Ideas]
I would love the Windows Terminal to be an easy ssh:// protocol handler. Its a bit awkward to have to write my own script and registry key for an implementation with Windows Terminal on a business device. 😅

**Comment by lhecker** [comment]
We haven't had a request like this before, so I think it's fair to say that it's somewhat unusual. 😅 But I can definitely see how this could be really useful. I'm not sure if it warrants adding official support for it though, unless it would be requested by more people.
